tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -64,6 +72,21 @@ Verify Service Is Available In The Explore Page
   Capture Page Screenshot
   Page Should Contain Element    //article//*[.='${app_name}']
 
+Remove Disabled Application From Enabled Page
+   [Documentation]  The keyword let you re-enable or remove the card from Enabled page
+   ...              for those application whose license is expired. You can control the action type
+   ...              by setting the "disable" argument to either "disable" or "enable".
+   [Arguments]  ${app_id}
+   ${card_disabled_xp}=  Set Variable  //article[@id='${app_id}']//div[contains(@class,'enabled-controls')]/span[contains(@class,'disabled-text')]
+   Wait Until Page Contains Element  xpath:${card_disabled_xp}  timeout=90
+   Click Element  xpath:${card_disabled_xp}
+   Wait Until Page Contains   To remove card click
+   ${buttons_here}=  Get WebElements    xpath://div[contains(@class,'popover__body')]//button[text()='here']
+   Click Element  ${buttons_here}[1]
```

Comment:
Are there multiple buttons with text "here"? I would try to find a different way to click them if so, the index of the correct button is prone to changing

Reply:
unfortunately there are two button with text "here" one after the other. I didn't find any attribute to use to capture the link differently. However, it probably gonna change because of a file issue which may change the UI. So, for now i would keep it as it is and after the changes are released I will change the logic here.